src/components/SwitchPlanPopup/SwitchPlanPopup.js

```diff
@@ -36,11 +36,13 @@ const SwitchPlanPopup = ({
   onCancel,
   onSwitchSuccess,
   isPartOfCancellationFlow,
+  switchAlgorithm,
   t
 }) => {
   const STEPS = {
     SWITCH_DETAILS: 'SWITCH_DETAILS',
-    CONFIRMATION: 'CONFIRMATION'
+    CONFIRMATION: 'CONFIRMATION',
+    ERROR: 'ERROR'
```

Comment:
Is this used anywhere?

Reply:
Yep, you're right. I deleted this line in the new commit